README.md

```diff
@@ -1,27 +1,31 @@
 # sync-db
 
-Database synchronization utility.
+Command line utility to synchronize and version control relational database objects across databases.
 
 [![oclif](https://img.shields.io/badge/cli-oclif-brightgreen.svg)](https://oclif.io)
 [![Version](https://img.shields.io/npm/v/@leapfrogtechnology/sync-db.svg)](https://npmjs.org/package/@leapfrogtechnology/sync-db)
 [![Downloads/week](https://img.shields.io/npm/dw/@leapfrogtechnology/sync-db.svg)](https://npmjs.org/package/@leapfrogtechnology/sync-db)
 [![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/laudio/sync-db.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/laudio/sync-db/context:javascript)
 [![License](https://img.shields.io/npm/l/@leapfrogtechnology/sync-db.svg)](https://github.com/kabirbaidhya/sync-db/blob/master/package.json)
 
-# Set up
+# Setup
 
-## Install sync-db
+## Installation of sync-db
 
 Install `sync-db` globally.
 
-    $ yarn global add @lftechnology/sync-db
+    $ yarn global add @leapfrogtechnology/sync-db
 
-Install database driver(s) of the database(s) that are to be synced.
+Globally, install node database driver(s) of the database(s) that are to be synced, for example this is how you would install driver mssql:
```

Comment:
install the mssql driver.

Reply:
Are we sure we want to make the end user install the package globally? Adding it to a existing project makes more sense.

Also, we have npx these days which makes global modules easier to run.

Reply:
Should I add documentation for both global and local alternatives?